backend/repository/data/client/baz_client_update.json

```diff
@@ -0,0 +1,33 @@
+{
+    "_comment_": "JSON for updating a tchannel client.",
```

Comment:
The JSON schema way is to name this field "description", http://json-schema.org/latest/json-schema-validation.html#rfc.section.7.2